src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -82,8 +81,11 @@ HttpStatusCode mongoSubscribeContext
     sub.append(CSUB_REFERENCE, requestP->reference.get());
 
     /* Throttling */
-    if (!requestP->throttling.isEmpty()) {
-      sub.append(CSUB_THROTTLING, (long long) requestP->throttling.parse());
+    long long throttling = 0;
```

Comment:
Yes, I need throttling later in this funcion, for the subCache.
That's why I "extract" the value as its own variable.